puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/rvm.groovy

```diff
@@ -20,7 +20,6 @@ def withRVM(commands, ruby = '2.0', name = '') {
 
     sh """#!/bin/bash -l
         rvm use ruby-${ruby}@${gemset(name)} --create
-        gem install bundler
```

Comment:
why moving this out?

Reply:
I moved this into a configureRVM script so it does it once. I was finding that in parallel, this can cause random failures.